### ISSUE TITLE
Need help getting pymisp, pydeep and lief working

### ISSUE THREAD

**Issue by jezkerwin** [MISP/MISP#2868]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Support
| OS version (server)       | RedHat 7.4
| OS version (client)       | 10,
| PHP version               |  7.1
| MISP version / git hash   | 2.4.86, 

When running the diagnostic tool, I'm getting the following error message for pymisp, pydeep and lief

```
 pymisp:… Please install pymisp: pip install pymisp
pydeep:… Please install pydeep: pip install git+https://github.com/kbandla/pydeep.git
magic:… OK
lief:… Please install lief, documentation here: https://github.com/lief-project/LIEF
```

I'm run through the install document and installed the packages as per the instructions as the root user.
When running python as the root user, the import of the modules works successfully.
When running as apache user the imports fail.

```
[root@misp01 ~]# python
Python 2.7.5 (default, May  3 2017, 07:55:04)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-14)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import lief
>>> import pymisp
WARNING [abstract.py:19 - <module>() ] You're using python 2, it is strongly recommended to use python >=3.5
WARNING [mispevent.py:26 - <module>() ] You're using python 2, it is strongly recommended to use python >=3.5
WARNING [api.py:31 - <module>() ] You're using python 2, it is strongly recommended to use python >=3.5
>>> import pydeep
```
```
[root@misp01 ~]# su -s /bin/bash apache -c "python"
Python 2.7.5 (default, May  3 2017, 07:55:04)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-14)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import lief
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named lief
>>> import pydeep
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named pydeep
>>> import pymisp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named pymisp
```

I've checked the sys.path variables between the two users and the only difference is the following path

`/usr/lib64/python2.7/site-packages/lief-0.8.3-py2.7-linux.egg`

Any thoughts or suggestions on what to check next?

Thanks for the help.

**Comment by jezkerwin** [comment]
I think I figured it out. the permissions on the site-package for pymisp are

`drwxr-x---.  4 root root   4096 Jan 23 20:20 pymisp`

I installed as the root user via 'sudo su -l ' so I don't understand how it's not working.
I installed on another system but that wasn't using sudo su -l as the root user and that worked so I'm wondering if because I didn't log in as the root user it messed the permissions up.
